### PR TITLE
Create profile card

### PR DESCRIPTION
**Pull request by evabrinkmann**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
+  const [profiles, setProfiles] = useState(
+    loadFromLocal('profiles') || userData
+  )
+
+  return (
+    <Router>
+      <AppGrid>
+        <Navigation />
+        <Switch>
+          <Route path="/profile-pool">
```
Comment (jamarob) [inline src/App.js:20]: Maybe use `<Route exact path={["/","/profile-pool"]}>` to link from url root to profile page as well?

File: src/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function Navigation() {
+  const [toggle, setToggle] = useState(true)
```
Comment (jamarob) [inline src/Navigation.js:6]: What's the purpose of this state? As far as I can tell, this doesn't do anything besides triggering the logging on click? Maybe you forgot to delete it?

File: src/StorybookStyles.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// import React from 'react'
```
Comment (jamarob) [inline src/StorybookStyles.js:1]: Delete comments instead of pushing them to master, if the file is empty delete it.

File: src/utils.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export function saveToLocal(name, data) {
```
Comment (jamarob) [inline src/utils.js:1]: As far as I can tell these functions are only used in `App.js` to save and load profiles. Maybe introduce a const with the save key and export simpler functions `loadProfilesFromLocal()` and `saveProfilesToLocal(profiles)` and keep the more generic ones local to this file?

File: src/stories/CardHead.stories.js [hunk 5]
```diff
@@ -1,13 +1,18 @@
 import React from 'react'
 import CardHead from '../components/CardHead'
 import userData from '../userData'
+import GlobalStyles from '../GlobalStyles'
```
Comment (jamarob) [inline src/stories/CardHead.stories.js:4]: Never used, maybe delete it?

File: src/components/Card.js [hunk 6]
```diff
@@ -24,15 +25,15 @@ export default function Card({ user, setActiveCard, isActive }) {
       <ProfileCardStyled onClick={() => activeCard()}>
```
Comment (jamarob) [inline src/components/Card.js:25]: Maybe `() => setActiveCard(isActive ? '' : user.id)` and delete `activeCard()`?

File: src/components/CardBody.js [hunk 7]
```diff
@@ -136,6 +136,28 @@ export default function CardBody({
...
+    }
+  }
+
+  ul {
+    line-height: 2.2;
+    list-style-type: none;
+  }
+
+  li {
+    list-style-type: none;
```
Comment (jamarob) [inline src/components/CardBody.js:159]: I think this is not needed on `<li>`s if the parent has `list-style: none`.

File: src/components/CardBody.js [hunk 8]
```diff
@@ -136,6 +136,28 @@ export default function CardBody({
...
+  a {
+    color: black;
+    text-decoration: none;
+
+    &:hover {
+      color: #549dd8;
+    }
+  }
+
+  ul {
```
Comment (jamarob) [inline src/components/CardBody.js:153]: I think you can do without the `<ul>`s and remove some nesting in your JSX.

File: src/components/CardList.js [hunk 9]
```diff
@@ -1,25 +1,39 @@
...
-        />
-      ))}
-    </>
+    <Scroller>
+      <StyledMain>
+        {profiles.map(user => (
+          <Card
+            key={user.id}
+            user={user}
+            {...user}
```
Comment (jamarob) [inline src/components/CardList.js:20]: User is passed down as a an object and used that way in `<Card>`, spreading it in again is not necessary, I think.

File: src/components/Header.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Header({ headerTitle }) {
```
Comment (jamarob) [inline src/components/Header.js:4]: Maybe rename to just `title`, imho no need to namespace with the component name.

File: src/components/firebase.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
+import firebase from 'firebase/app'
```
Comment (jamarob) [inline src/components/firebase.js:1]: I am not shure that you should push this file to github? You are not using firebase atm, but doesn't this leak sensitive information?

**Review by jamarob — APPROVED** [review]
If you want to use firebase, research which of your firebase files belong into github!

Apart from that, the code is very well written and clean! I like it!

File: src/components/firebase.js (hunk 11 above)
Comment (evabrinkmann) [reply src/components/firebase.js:1]: You are right, I will folllow up on this one!

**Pull request merged**